aten/src/ATen/native/native_functions.yaml

```diff
@@ -11837,20 +11837,13 @@
 - func: linalg_qr(Tensor A, str mode='reduced') -> (Tensor Q, Tensor R)
   python_module: linalg
   variants: function
-  dispatch:
-    CompositeExplicitAutograd: linalg_qr
+  structured_delegate: linalg_qr.Q
 
-- func: linalg_qr.out(Tensor A, str mode='reduced', *, Tensor(a!) Q, Tensor(b!) R) -> (Tensor(a!) Q, Tensor(b!) R)
+- func: linalg_qr.Q(Tensor A, str mode='reduced', *, Tensor(a!) Q, Tensor(b!) R) -> (Tensor(a!) Q, Tensor(b!) R)
```

Comment:
I thought out overloads needed to have the name of the first output. But sure, reverting this change.

Reply:
Initial naming can be anything, it's a convention that if there are multiple outputs the name of the first output is used. It was overlooked initially that it was using ".out", but we shouldn't change it now. Though I'm not sure how exactly the rename would break things.